tests/runner/test_runner.py

```diff
@@ -48,6 +48,13 @@
 from prefect.utilities.filesystem import tmpchdir
 
 
+def make_minimal_deployment_schedule(**kwargs) -> MinimalDeploymentSchedule:
```

Comment:
was it intentional to include these changes?

Reply:
yeah, temporarily. i was confused to see the failures due to unrelated deprecation warnings

Reply:
now that thats merged, I will revert this extra commit here